images/openshift-proxy-pull-test.yml

```diff
@@ -0,0 +1,22 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/machine-config-operator
+content:
+  source:
+    dockerfile: Dockerfile.proxy
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/machine-config-operator.git
+enabled_repos:
```

Comment:
https://github.com/openshift/machine-config-operator/blob/68e602d88967f30eebf86086d68f7c6303064893/Dockerfile.proxy doesn't use any rpms. Please remove the `enabled_repos` field.